src/main/java/com/nerodesk/om/aws/AwsDoc.java

```diff
@@ -120,6 +120,11 @@ public void read(@NotNull final OutputStream output) throws IOException {
     @Override
     public void write(final InputStream input, final long size)
         throws IOException {
+        if (size == 0) {
```

Comment:
@dmzaytsev as you said before, theoretically, we might decide to save doc with zero length in some cases.
This can be controlled by adding SafeDoc decorator or not